### DISCUSSION TITLE
Display File Renderer

### DISCUSSION THREAD

**Discussion by thomprl** [codefori/vscode-ibmi#2000] [Q&A]
I have the IBMi Renderer Extension installed but how do I open the rendered screen in VSCode?  I searched the online docs and don't see it mentioned.  Might be nice to add it to the docs if there is some trick to it.

**Comment by thomprl** [comment]
Never mind, if it had been a snake it would've bit me.

![image](https://github.com/codefori/vscode-ibmi/assets/17936568/81f0ed6b-8cfe-4f00-b180-fa8758f4952d)

**Comment by worksofliam** [comment]
I believe we have an issue open to create a page in our docs for the renderer extension.
